examples/multiphase_flow/ex0/example.cpp

```diff
@@ -178,30 +189,39 @@ run_example(int argc, char* argv[])
 #endif
 
         const string& ls_name = "level_set";
-
         Pointer<CellVariable<NDIM, double> > phi_var = new CellVariable<NDIM, double>(ls_name);
         adv_diff_integrator->registerTransportedQuantity(phi_var);
         adv_diff_integrator->setDiffusionCoefficient(phi_var, 0.0);
+        // Set the advection velocity of the bubble.
         adv_diff_integrator->setAdvectionVelocity(phi_var, time_integrator->getAdvectionVelocityVariable());
 
-        // Set options for resetting level set data
-         Pointer<RelaxationLSMethod> level_set_ops =
+        Pointer<RelaxationLSMethod> level_set_ops =
             new RelaxationLSMethod("RelaxationLSMethod", app_initializer->getComponentDatabase("RelaxationLSMethod"));
         LSLocateCircularInterface* ptr_LSLocateCircularInterface =
             new LSLocateCircularInterface("LSLocateCircularInterface", adv_diff_integrator, phi_var, circle);
         level_set_ops->registerInterfaceNeighborhoodLocatingFcn(&callLSLocateCircularInterfaceCallbackFunction,
                                                                 static_cast<void*>(ptr_LSLocateCircularInterface));
-
         SetLSProperties* ptr_SetLSProperties = new SetLSProperties("SetLSProperties", level_set_ops);
         adv_diff_integrator->registerResetFunction(phi_var, &callSetLSCallbackFunction, static_cast<void*>(ptr_SetLSProperties));
 
-        // Setup the advected and diffused quantities.
-        Pointer<CellVariable<NDIM, double> > rho_var = new CellVariable<NDIM, double>("rho");
-        adv_diff_integrator->registerTransportedQuantity(rho_var);
-        adv_diff_integrator->setDiffusionCoefficient(rho_var, 0.0);
+        // Setup the INS maintained material properties.
+        Pointer<Variable<NDIM> > rho_var;
+        if (conservative_form)
+        {
+            rho_var = new SideVariable<NDIM, double>("rho");
+        }
+        else if (!conservative_form)
+        {
+            rho_var = new CellVariable<NDIM, double>("rho");
+        }
+        else
```

Comment:
No need for `else if` if `conservative_form` is a `bool` -- it can just be an `else`.